node/package.json

```diff
@@ -8,7 +8,7 @@
     "format": "prettier --write '**/*.{js,jsx,json,ts,tsx}'",
     "lint:fix": "eslint src --ext ts --ext js --fix",
     "start:dev": "export NODE_PATH=$(pwd)/src && export NODE_ENV=dev && export ENV=dev && ts-node src/server.ts",
-    "test": "export ENV=test && export NODE_ENV=test && npx mocha -r tsconfig-paths/register -r ts-node/register -r source-map-support/register './test/**/*.{spec,test}.{js,ts}' --timeout 5000 --full-trace --recursive --reporter spec",
+    "test": "export NODE_PATH=$(pwd)/src ENV=test && export NODE_ENV=test && npx mocha -r tsconfig-paths/register -r ts-node/register -r source-map-support/register './test/**/*.{spec,test}.{js,ts}' --timeout 5000 --full-trace --recursive --reporter spec",
```

Comment:
question: if we switch to [ts-mocha](https://www.npmjs.com/package/ts-mocha) can we simplify (i.e. maybe we won't need to set `NODE_PATH` cause tsconfig.json will provide it?)